docs/AboutUs.adoc

```diff
@@ -4,53 +4,60 @@ ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 :imagesDir: images
 :stylesDir: stylesheets
 
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-The dummy content given below serves as a placeholder to be used by future forks of the project. +
+PocketBook V1.1 was developed by the https://github.com/orgs/CS2103AUG2017-F09-B3/teams/developers/members[F09-B3]
```

Comment:
Just saying `Pocketbook` instead of `PocketBook V1.1` is fine. This will relieve you some burden of updating this number every week.